web-tier/src/main/java/com/outdoor/foodcalc/endpoint/DishApi.java

```diff
@@ -56,7 +56,7 @@ DishView getDish(@ApiParam(value =  "ID of the dish", required = true)
     @PostMapping(consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.CREATED)
     SimpleDish addDish(@ApiParam(value = "Dish", required = true)
-                       @RequestBody @Valid SimpleDish dish);
+                       @RequestBody SimpleDish dish);
```

Comment:
Could you please remind me the reason to remove Valid annotation?

Reply:
It is used in DishEndpoint